vr-wordle/src/Components/GameEnd.js

```diff
@@ -8,6 +8,8 @@ export default function GameEnd(props) {
     if (session) {
       session.end();
       props.endGame();
+    } else if (process.env.NODE_ENV === 'development') {
+      props.endGame();
     }
```

Comment:
Really like this fix, will make testing on local a lot easier. Definitely won't cause issues on the deploy right?

Reply:
Shouldn't do!